### PR TITLE
chores: Use new standalone account

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

File: ci-operator/step-registry/redhat-performance/backstage-performance/redhat-performance-backstage-performance-workflow.yaml [hunk 1]
```diff
@@ -5,7 +5,8 @@ workflow:
       COMPUTE_NODE_REPLICAS: "5"
       COMPUTE_NODE_TYPE: m6a.2xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6a.2xlarge
-      BASE_DOMAIN: aws.rhperfscale.org
+      BASE_DOMAIN: rhdh-perfscale.devcluster.openshift.com
```
Comment (pmacik) [inline ci-operator/step-registry/redhat-performance/backstage-performance/redhat-performance-backstage-performance-workflow.yaml:8]: @jhutar Please update also scalability workflow [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-performance/backstage-performance/scalability/redhat-performance-backstage-performance-scalability-workflow.yaml#L8)

**Review by pmacik — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/redhat-performance/backstage-performance/redhat-performance-backstage-performance-workflow.yaml (hunk 1 above)
Comment (jhutar) [reply ci-operator/step-registry/redhat-performance/backstage-performance/redhat-performance-backstage-performance-workflow.yaml:8]: I see, added, thanks.

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-redhat-performance-backstage-performance-main-mvp

**Comment by pmacik** [comment]
/pj-rehearse pull-ci-redhat-performance-backstage-performance-main-mvp

**Review by pmacik — APPROVED** [review]
/lgtm
/approve

**Pull request merged**